OutSystems/outsystems-datagrid-reactive
# ROU-3934: Add client action to remove rows by number or row key

This PR is for add new RemoveRows client action

### What was done
* Implemented new removeRowsByNumberOrKey method for row feature
* Added new RemoveRows API method.

### Test Steps
Test 1:
    1. Go to the test page.
    2. Choose row numbers 0 and 1 in the dropdown.
    3. Choose row keys "Black and grey pro laptop" and "Black and silver headphones" to remove in the dropdown.
    4. Click in RemoveRows button.
    Expected: Only rows 0 and 1 have been removed.

Test 2:
    1. Go to the test page.
    3. Choose row keys "Black and grey pro laptop" and "Black and silver headphones" in the dropdown.
    4. Click in RemoveRows button.
    Expected: Only rows with keys "Black and grey pro laptop" and "Black and silver headphones" have been removed.

Test 3:
    1. Go to the test page.
    2. Click in RemoveRows button.
    Expected: A feedback error message with the text "GRID-API-02011: Rows list is empty." appeared.

Test 4:
    1. Go to the test page.
    2. Choose row number -1 in the dropdown.
    3. Click in RemoveRows button.
    Expected: A feedback error message with the text "GRID-API-02011: Row '-1' not found." appeared.

Test 5:
    1. Go to the test page.
    2. Choose row key "Invalid key" in the dropdown.
    3. Click in RemoveRows button.
    Expected: A feedback error message with the text "GRID-API-02011: Row 'Invalid key' not found." appeared.


### Checklist
